MultiplayerModFiles/Launcher/RunGame.py

```diff
@@ -385,25 +372,20 @@ def DeleteUnusedDlcs(gamepath):
             # make sure it's a folder
             if os.path.isdir(gamepath + nf + file):
                 # if inside the folder there is a file called "32playermod.identifier" delete this folder
-                FoundDlc = False
                 if "32playermod.identifier" in os.listdir(gamepath + nf + file):
                     Log("Found OLD DLC: " + file)
                     # delete the folder even if it's not empty
                     # Windows command
                     if (iow):
                         command = "rmdir /S /Q \"" + gamepath + nf + file + "\""
-                        Log("Command: " + command)
                         os.system(command)
-                        FoundDlc = file
-                        Log("Deleted OLD DLC: " + file)
                     # Linux command
-                    if (iol):
+                    elif (iol):
                         command = "rm -r \"" + gamepath + nf + file + "\""
-                        Log("Command: " + command)
                         os.system(command)
-                        FoundDlc = file
-                        Log("Deleted OLD DLC: " + file)
-    return FoundDlc
```

Comment:
I have no idea why this even exists

1. it's defined as a bool but never used as such
2. it's set later to a file name instead of a bool (just the name not even the path)
3. it's returned but no function makes a use of it

was this you kyle....